application/controllers/timers.py

```diff
@@ -0,0 +1,90 @@
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
+from application.models.timer import Timer
+from application import db
+
+timer_fields = {
+    'id': fields.Integer,
+    'work_interval': fields.String,
+    'rest_interval': fields.String
+}
```

Comment:
We landed on using this exact same naming with `workInterval` and `breakInterval`. So neat and bewildering to see how this is all working.